Doc/howto/perf_profiling.rst

```diff
@@ -210,31 +210,57 @@ of ``perf``.
 How to work without frame pointers
 ----------------------------------
 
-If you are working with a Python interpreter that has been compiled without frame pointers
-you can still use the ``perf`` profiler but the overhead will be a bit higher because Python
-needs to generate unwinding information for every Python function call on the fly. Additionally,
-``perf`` will take more time to process the data because it will need to use the DWARF debugging
-information to unwind the stack and this is a slow process.
+If you are working with a Python interpreter that has been compiled without
+frame pointers you can still use the ``perf`` profiler but the overhead will be
+a bit higher because Python needs to generate unwinding information for every
+Python function call on the fly. Additionally, ``perf`` will take more time to
+process the data because it will need to use the DWARF debugging information to
+unwind the stack and this is a slow process.
 
-To enable this mode, you can use the environment variable :envvar:`PYTHON_PERF_JIT_SUPPORT` or the
-:option:`-X perf_jit <-X>` option, which will enable the JIT mode for the ``perf`` profiler.
+To enable this mode, you can use the environment variable
+:envvar:`PYTHON_PERF_JIT_SUPPORT` or the :option:`-X perf_jit <-X>` option,
+which will enable the JIT mode for the ``perf`` profiler.
 
-When using the perf JIT mode, you need an extra step before you can run ``perf report``. You need to
-call the ``perf inject`` command to inject the JIT information into the ``perf.data`` file.
+.. note::
+
+    Due to a bug in the ``perf`` tool, only ``perf`` versions higher than v6.8
+    will work with the JIT mode.  The fix was also backported to the v6.7.2
+    version of the tool. 
```

Comment:
```suggestion
    version of the tool.
```
Removed whitespace in the end. In theory, it should pass [linter test](https://github.com/python/cpython/actions/runs/8986576592/job/24683019918?pr=118708#step:4:159) now.